### PR TITLE
Add timestamp and a POST endpoint

### PR DESCRIPTION
**Pull request by rlustin**
Add a timestamp to Translation model and a POST endpoint to API.

The new POST endpoint expects `key`, `locale` and `content` parameters and returns the persisted Translation.

Example:
```bash
curl -H "Content-Type: application/json" -X POST -d '{"key": "hey.you", "locale":" fr", "content": "Hey toi !" }' http://localhost:3000/api/translations

{"id":12,"key":"hey.you","locale":"fr","content":"Hey toi !","timestamp":"2016-11-06 21:57:54"}
```

@Signez, could you please review?

### CODE REVIEW

File: migrations/20161029110638_create_translations/up.sql [hunk 1]
```diff
@@ -2,7 +2,8 @@ CREATE TABLE translations (
   id INTEGER PRIMARY KEY NOT NULL,
   key TEXT NOT NULL,
   locale TEXT NOT NULL,
-  content TEXT
+  content TEXT,
+  timestamp TEXT NOT NULL DEFAULT CURRENT_TIMESTAMP
```
Comment (Signez) [inline migrations/20161029110638_create_translations/up.sql]: Shouldn't that be on another migration? Modifying old migrations is not a very good practice, but I don't know if we should care a that point of the development. Just a head's up :wink:

File: src/api.rs [hunk 2]
```diff
@@ -4,23 +4,24 @@ extern crate rustc_serialize;
...
-    use schema::translations::dsl::*;
+pub fn translations_index(_: &mut Request) -> IronResult<Response> {
     use std::collections::HashMap;
 
     let connection = database::establish_connection();
-    let results = translations.filter(sql("id IN (SELECT MAX(id) FROM translations GROUP BY key, locale)"))
-        .load::<Translation>(&connection)
+    let results = translations.filter(
+        sql("(key, locale, timestamp) IN (SELECT key, locale, MAX(timestamp) FROM translations GROUP BY key, locale)")
+    ).load::<Translation>(&connection)
```
Comment (Signez) [inline src/api.rs]: This identation makes me uneasy. Could you had a level of indentation to both this line?

File: migrations/20161029110638_create_translations/up.sql (hunk 1 above)
Comment (rlustin) [inline migrations/20161029110638_create_translations/up.sql]: You are definitely true. I think we just shouldn’t care for now 😄

File: src/api.rs (hunk 2 above)
Comment (rlustin) [inline src/api.rs]: Sure 👍

File: migrations/20161029110638_create_translations/up.sql (hunk 1 above)
Comment (Signez) [inline migrations/20161029110638_create_translations/up.sql]: That's fine for me :+1:

**Pull request merged**